app/tests/spec/views/sign_in_unblock.js

```diff
@@ -217,6 +217,26 @@ define(function (require, exports, module) {
         });
       });
 
+      describe('incorrect unblock_code', () => {
+        const incorrectUnblockCodeError =
+          AuthErrors.toError('INCORRECT_UNBLOCK_CODE');
+
+        beforeEach(() => {
+          sinon.stub(view, 'signIn').callsFake(() => Promise.reject(incorrectUnblockCodeError));
+
+          sinon.stub(view, 'navigate').callsFake(() => Promise.resolve());
```

Comment:
You can remove this statement and replace it with a `sinon.spy` for `showValidationError`. Ex: https://github.com/mozilla/fxa-content-server/blob/1b6bec275515f9125702a1832befc037a044367d/app/tests/spec/views/sign_in_unblock.js#L139